packages/api/lambdas/bulk-operation.js

```diff
@@ -40,50 +50,6 @@ async function applyWorkflowToGranules({
   return await Promise.all(applyWorkflowRequests);
 }
 
-/**
```

Comment:
This was commented-out. I presume it's no longer relevant?

Reply:
This method was being used as a default in `deleteGranuleAndFiles`, which originally assumes that pgGranule might not be set and attempts to do a lookup using the looked up (by granule ID) dynamo granule's collection and garnuleId. 

It's no longer needed in the context of bulk-operations because of the switch to reading the postgres granule -- and because we haven't settled on an approach to dealing with the collection/granuleId queries for the pg database, we're not doing this sort of query *yet*.